metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/dao/WikidataAccessDao.java

```diff
@@ -197,25 +190,24 @@ private Resource getModelFromSPARQL(String uri) {
    * @return true if the retrieved resource is duplicated
    */
   private boolean isDuplicate(Resource resource) {
-    return (resource.hasProperty(OWL.sameAs) && !resource.hasProperty(RDFS.label));
+    return (resource!= null && resource.hasProperty(OWL.sameAs) && !resource.hasProperty(RDFS.label));
   }
 
-  private Resource fetchFromSPARQL(String uri) {
+  private Resource fetchFromSPARQL(String uri) throws WikidataAccessException {
     String sDescribe = "DESCRIBE <" + uri + ">";
 
     Model m = ModelFactory.createDefaultModel();
     QueryEngineHTTP endpoint = new QueryEngineHTTP(SPARQL, sDescribe);
     try {
       return endpoint.execDescribe(m).getResource(uri);
-    } catch (RiotException e) {
-      LOGGER.error("Interrupted while querying Wikidata from the WikidataAccessDao", e);
+    } catch (Exception e) {
+      //LOGGER.error("Interrupted while querying Wikidata from the WikidataAccessDao", e);
```

Comment:
No need to log anymore?

Reply:
the exception is now converted and retrown, logging is needed only when the exception is really handled. I remove the commented code now